internal/data.go

```diff
@@ -24,6 +24,10 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
+var (
+	UserTableKeyPrefix = []byte("data")
```

Comment:
I think we should log DDLs too, but we can add it later also.

Reply:
DDLs will automatically get skipped with this approach because this prefix is added for any operation performed on the collection.